### DISCUSSION TITLE
Refreshing user_metadata after updateUserById

### DISCUSSION THREAD

**Discussion by ntkris** [supabase/supabase#10592] [Questions]
I'm updating information for a user on the backend using `updateUserById`:

```
        await serviceSupabase.auth.admin.updateUserById(fetchOrg.user_id, {
          user_metadata: finalData,
        });
```

These changes are not immediately reflected on the `user` object on the frontend. 

How can I fetch the latest information for the user?

**Comment by GaryAustin1** [comment]
auth.getUser() probably.  It goes to the database to update the user info.
